LRAtsSDKSample/Examples/GetEnvelopeViewController.swift

```diff
@@ -26,11 +26,11 @@ class GetEnvelopeViewController: UIViewController {
     
     private func configureLRAtsExample() {
         Task {
-            guard let appID = appIDTextField.text else { return }
-            // You should provide your appID here
```

Comment:
These are previous changes made in develop by levi9 team which were not merged to main